go-controller/pkg/util/pod_annotation.go

```diff
@@ -443,19 +443,13 @@ func IsValidPodAnnotation(podAnnotation *PodAnnotation) bool {
 	return podAnnotation != nil && len(podAnnotation.MAC) > 0
 }
 
-func joinSubnetToRoute(isIPv6 bool, gatewayIP net.IP) PodRoute {
-	joinSubnet := config.Gateway.V4JoinSubnet
+func joinSubnetToRoute(netinfo NetInfo, isIPv6 bool, gatewayIP net.IP) PodRoute {
+	joinSubnet := netinfo.JoinSubnetV4()
 	if isIPv6 {
-		joinSubnet = config.Gateway.V6JoinSubnet
+		joinSubnet = netinfo.JoinSubnetV6()
```

Comment:
I think not.. but let me double check to ensure singlestack v6 works

Reply:
it looks good:
```
/ # ip -6 r
2010:100:200:1::/64 dev ovn-udn1 proto kernel metric 256 pref medium
2010:100:200::/60 via 2010:100:200:1::1 dev ovn-udn1 metric 1024 pref medium
fd00:10:96::/112 via 2010:100:200:1::1 dev ovn-udn1 metric 1024 pref medium
fd00:10:244:2::/64 dev eth0 proto kernel metric 256 pref medium
fd00:10:244::/48 via fd00:10:244:2::1 dev eth0 metric 1024 pref medium
fd98::/64 via fd00:10:244:2::1 dev eth0 metric 1024 pref medium
fd99::/64 via 2010:100:200:1::1 dev ovn-udn1 metric 1024 pref medium ===> this one
fe80::/64 dev eth0 proto kernel metric 256 pref medium
fe80::/64 dev ovn-udn1 proto kernel metric 256 pref medium
default via 2010:100:200:1::1 dev ovn-udn1 metric 1024 pref medium
```
but this is good I checked because I missed a spot :)